src/resources/views/filters/export.blade.php

```diff
@@ -0,0 +1,89 @@
+<li
+        class="{{ $filter->name }}-class btn btn-default"
+        style="background-color: seagreen; color: white;   height: 32px;line-height: 17px;margin: 0px 10px; "
+        name="{{ $filter->name }}"
+        id="export-button"
+>Exel Export</li>
+
+<li>
+    <a href="#"
+       class="btn btn-default"
+       style="display: none; background-color: lightgrey; color: black;  margin: 0px 10px; height: 32px; line-height: 17px;"
+       id="export-button-download"
+    >Download</a>
```

Comment:
i18n

Reply:
do I need to translate in all languages?

Reply:
Only `en` - it's quite probably there's already a "download" and "loading" one about the place.